README.md

```diff
@@ -470,7 +470,7 @@ You will need access to the prompt flow files for this experience, since we'll b
 
     The following code will create a folder named 'dist' with a Dockerfile and all the required flow files.
 
-TODO (P1 Jon): Stopped here. THere are two likely TODOs in the acr.bicep file to help make the following work.
+TODO (P1 Jen ): Stopped here. THere are two likely TODOs in the acr.bicep file to help make the following work.

```

Comment:
This is still part of your todo, I didn't see the `az acr build -t $FULL_IMAGE_NAME -r $NAME_OF_ACR .` command line change.  Don't you need to use the name of the agent as a parameter?